daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -902,8 +874,8 @@ private[lf] final class Compiler(
   private[this] def translateCommand(env: Env, cmd: Command): s.SExpr = cmd match {
     case Command.Create(templateId, argument) =>
       t.CreateDefRef(templateId)(s.SEValue(argument))
-    case Command.CreateByInterface(interfaceId, templateId, argument) =>
-      t.CreateByInterfaceDefRef(templateId, interfaceId)(s.SEValue(argument))
+    case Command.CreateByInterface(interfaceId @ _, templateId @ _, argument) =>
+      SBResolveCreate(s.SEValue(argument))
```

Comment:
Isn't `argument` of type `templateId`? I don't think `SBResolveCreate(_)` makes sense with a template argument.

Reply:
Good catch. I will drop the `ByInterface` commands first, and rebase this PR on top of. 
This will make the problem disappear.

Reply:
The command is gone.